site/src/xServices/auth/authXService.ts

```diff
@@ -469,24 +511,27 @@ export const authMachine =
         signIn: async (_, event) => {
           return await API.login(event.email, event.password)
         },
-        signOut: async () => {
+        checkHost: async () => {
+          const appHost = await API.getApplicationsHost()
+          return appHost.host
```

Comment:
I wish I could just `checkHost: API.getApplicationsHost`, but XState won't allow me to type event correctly in the guard on ln 648.